### PR TITLE
 Add a way to get all unique and virtual accounts from the economy service

### LINKED ISSUE #1398: Can't get list of accounts from EconomyService
The service lets me get/create accounts and check if accounts exist, but doesn't let me get a list of all unique and virtual accounts.

It should.

I need access to arbitrary virtual accounts, but i don't know what they are, which means i can't tab complete their identifiers.

### PR DESCRIPTION
**Pull request by ImMorpheus**
Fix https://github.com/SpongePowered/SpongeAPI/issues/1398

### CODE REVIEW

File: src/main/java/org/spongepowered/api/service/economy/EconomyService.java [hunk 1]
```diff
@@ -108,6 +109,20 @@
      */
     Optional<Account> getOrCreateAccount(String identifier);
 
+    /**
+     * Returns a {@link Stream} of all stored {@link UniqueAccount} objects.
+     *
+     * @return A {@link Stream} of all stored {@link UniqueAccount} objects.
+     */
+    Stream<UniqueAccount> uniqueAccounts();
```
Comment (Zidane) [inline src/main/java/org/spongepowered/api/service/economy/EconomyService.java]: Add Stream and Collection getters.

I like Streams but we should not shoehorn people into doing them as a default. I made the same change for the CatalogRegistry.

Comment (ImMorpheus) [reply src/main/java/org/spongepowered/api/service/economy/EconomyService.java]: Done

Comment (Zidane) [reply src/main/java/org/spongepowered/api/service/economy/EconomyService.java]: Maybe also use the same phrasing in the CatalogRegistry that I've seen in other libraries?

https://github.com/SpongePowered/SpongeAPI/blob/api-8/src/main/java/org/spongepowered/api/registry/CatalogRegistry.java#L96

Comment (Zidane) [reply src/main/java/org/spongepowered/api/service/economy/EconomyService.java]: @ImMorpheus I meant the method names.

Comment (ImMorpheus) [reply src/main/java/org/spongepowered/api/service/economy/EconomyService.java]: Oops, done

**Pull request merged**